guide/payments/receive.md

```diff
@@ -0,0 +1,219 @@
+   alt-text = "Screen showing the invoice overlayed with a share sheet"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+</div>
+
+
+**Text**
+
+Sharing payment details as text is as straightforward as copying and pasting. It is also typical for Bitcoin applications to provide the receiver with a “share” button - allowing them to conveniently share addresses or payment links via other applications, with “share sheets” being offered by most mobile operating systems.
+
+**QR Codes**
+
+QR Codes are also a common way to exchange payment details when the two devices are near one another. It is not without its faults though, as some people may not be familiar with QR Codes, and lower-end phones may have difficulty reading them.
+
+Here are some constraints to consider when displaying QR Codes:
+
+- Complex QR Codes can be difficult for devices with low-resolution cameras to read.
+- Use high contrast between the QR Code blocks and their background.
+- Display the QR Code at the largest size so that the scanner can detect it easily.
+
+_For more technical details about optimizing QR Codes, [see this article](https://bitcoinops.org/en/bech32-sending-support/#creating-more-efficient-qr-codes-with-bech32-addresses)._
+
+## Awaiting the payment
+Once the receiver's payment details have been shared, senders still have to create and broadcast their transaction, leaving room for uncertainty as to when the payment will be completed and received. Think about how to keep the receiver informed whilst they await their payment.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/receive-with-pending.png"
+   retina = "/assets/images/guide/payments/receive/receive-with-pending@2x.png"
+   alt-text = "Screen showing a list of incoming payments"
+   caption = "By labeling addresses and storing them, you will be able to indicate the state of incoming payments."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/receive-pending-details.png"
+   retina = "/assets/images/guide/payments/receive/receive-pending-details@2x.png"
+   alt-text = "Screen showing details of an incoming payment that is pending"
+   caption = "Consider some users who are new to Bitcoin may not know how its transactions get processed."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+If your application has a section where the user can see a list of their payment requests or transaction history, this would be a good place to provide the user with updates on the payment’s status and what stage of completion it is at.
+
+Then once the payment has been finalized, you should also consider what the receiver may want to do with those funds. You may want to help facilitate those follow-up activities for example, moving the funds to a shared multi-key wallet or doing a [coinjoin]({{ '/guide/glossary/#coinjoin' | relative_url }}).
```

Comment:
```suggestion
Once the payment has been finalized, consider what the receiver may want to do with those funds. You may want to help facilitate those follow-up activities, for example moving the funds to a shared multi-key wallet or doing a [coinjoin]({{ '/guide/glossary/#coinjoin' | relative_url }}).
```

Reply:
The part about follow-up actions seem very hard to predict. But I'm not opposed to keeping it in here for now.